### ISSUE TITLE
Windows: Error when resolving relative deployments path from HardhatContext

### ISSUE THREAD

**Issue by RobertoSnap** [symfoni/hardhat-react-boilerplate#1]
![image](https://user-images.githubusercontent.com/4772930/98737102-088e0300-23a6-11eb-8466-baf99d196075.png)

packages/hardhat-react/src/TsMorhProject.ts:95 

```ts
   const relativeDeploymentsPath = path.relative(
      this.hre.config.paths.react,
      this.hre.config.paths.deployments + "/" + currentNetwork
    );
```
packages/hardhat-react/src/TsMorhProject.ts:179
```ts
      deploymentFile: deploymentFile
            ? `${relativeDeploymentsPath}/${deploymentFile}`
            : undefined,
```
Does not work in Windows.

Need to try and create a fix and run it on windows.

Fix needs to be created in 
https://github.com/symfoni/hardhat-plugins

**Comment by RobertoSnap** [comment]
@ramvi

**Comment by RobertoSnap** [comment]
Test version 0.0.46-alpha.0

**Comment by ramvi** [comment]
no go

**Comment by Datakunskap** [comment]
HELP!!!!

**Comment by RobertoSnap** [comment]
Fixed in version 0.1.7 with commit https://github.com/symfoni/hardhat-plugins/commit/5d134550e2e464fb1d470353f08eff13bb640ba1

Can you try with this version @DrScatman @ramvi

**Comment by ramvi** [comment]
Works perfectly!
